cli/flox/doc/include/environment-options.md

```diff
@@ -1,7 +1,8 @@
 ## Environment Options
 
-If no environment is specified for an environment command, the environment in
-the current directory or the last activated environment is used.
+If no environment is specified for an environment command,
+the environment in the current directory
+or the active environment that was last activated is used.
```

Comment:
What is the difference between those two?

Reply:
I think technically the first one ("last activated environment") could no longer be active i.e. it's the most recent activation it's no longer active. The second one ("activate environment that was last activated") is guaranteed to be still active.